test/k8sT/Updates.go

```diff
@@ -546,6 +565,8 @@ func InstallAndValidateCiliumUpgrades(kubectl *helpers.Kubectl, oldHelmChartVers
 		nbMissedTailCalls, err = kubectl.CountMissedTailCalls()
 		ExpectWithOffset(1, err).Should(BeNil(), "Failed to retrieve number of missed tail calls")
 		ExpectWithOffset(1, nbMissedTailCalls).To(BeNumerically("==", 0))
+
+		Expect(int(persistenConnEC.Load())).Should(Equal(0), "persistent tcp connection died in reload")
```

Comment:
This should be an `ExpectWithOffset`.

Reply:
Also, how do we know at this point that the `cmd` has exited? This is kinda related to the zero-valued atomic var. This seems like this test will produce false positives/negatives.